biothings/web/query/formatter.py

```diff
@@ -174,6 +184,11 @@ def transform(self, response, **options):
             template_miss = options.pop('template_miss', dict(found=False))
             responses = [self.transform(res, **options) for res in response]
             for tpl, res in zip(templates, responses):
+                total = res.get('total', {}).get('value') or 0
+                if tpl['query'] not in count_by_queries:
+                    count_by_queries[tpl['query']] = 0
+                count_by_queries[tpl['query']] += total
```

Comment:
@sengineer0 since we now just need to get the maximum value of `total`, we don't need to keep the `count_by_queryies` to store all total values, right?

Same for `count_query_exceed_max_size`.

Reply:
Good idea, I will change the code to reflect your recommendation.Thank you :)

Reply:
It's updated.
I am checking the request sent to ES to determine the default size, and will update the max_size if needed later. :)
@newgene